content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,129 +1,103 @@
-[Git-LFS / Git-annex](/doc/user-guide/related-technologies) and to ad-hoc
-scripts used to manage ML <abbr>artifacts</abbr> (training data, models, etc.)
-on cloud storage. DVC doesn't require special services, and works with
-on-premises storage (e.g. SSH, NAS) as well as any major cloud storage provider
-(Amazon S3, Microsoft Azure, Google Drive,
-[among others](/doc/command-reference/remote/add#supported-storage-types)).
-
-> For hands-on experience, we recommend following the
-> [versioning tutorial](/doc/use-cases/versioning-data-and-model-files).
-
-## DVC is not Git!
-
-DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to track
-data files and directories (among other purposes). They point to specific data
-contents in the <abbr>cache</abbr>, providing the ability to store multiple data
-versions out-of-the-box.
-
-Full-fledged
-[version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control)
-is left for Git and its hosting platforms (e.g. GitHub, GitLab) to handle. These
-are designed for source code management (SCM) however, and thus ill-equipped to
-support data science needs. That's where DVC comes in: with its built-in data
-<abbr>cache</abbr>, reproducible [pipelines](/doc/start/data-pipelines), among
-several other novel features (see [Get Started](/doc/start/) for a primer.)
-
-## Track data and models for versioning
-
-Let's say you have an empty <abbr>DVC repository</abbr> and put a dataset of
-images in the `images/` directory. You can start tracking it with `dvc add`.
-This generate a `.dvc` file, which can be committed to Git in order to save the
-project's version:
-
-```dvc
-$ ls images/
-0001.jpg 0002.jpg 0003.jpg 0004.jpg ...
-
-$ dvc add images/
-
-$ git add images.dvc .gitignore
-$ git commit -m "Track images dataset with DVC."
-```
-
-DVC's also allows to define the processes that build artifacts based on tracked
-data, such as an ML model, by writing a simple `dvc.yaml` file that connects the
-pieces together:
-
-> `dvc.yaml` files can be written manually or generated with `dvc run`.
-
-```yaml
-stages:
-  train:
-    cmd: python train.py images/
-    deps:
-      - images
-    outs:
-      - model.pkl
-```
-
-> See [Data Pipelines](/doc/start/data-pipelines) for a comprehensive intro to
-> this feature.
-
-`dvc repro` can now execute the `train` stage for you. DVC will track all of its
-outputs (`outs`) automatically. Let's do that, and commit this project version:
+[Version control](https://en.wikipedia.org/wiki/Version_control) has become a
+staple in software engineering because it allows effective collaboration on
+source code. This means having a change history to traverse (commits),
+developing parallel features (branching and merging), peer-reviews (pull
+requests), release management, etc. Imagine using these features for data
+engineering and machine learning!
+
+Unfortunately, versioning tools like [Git](https://git-scm.com/) are designed to
```

Comment:
Before we jump into motivation behind DVC (Git does not work) ... we should go into some of the actual advantages of versioning/tracking data/models in ML workflow. Some of the advantages from below should in some way here.

Reply:
OK, I made the first paragraph a little shorter and added a 2nd one about this (and an image placeholder after that). The motivation comes after that now. Not 100% sure about the summary of advantages though, it may be too abstract or redundant.

Reply:
This is a short bullet list now.